src/iiiftilesource.js

```diff
@@ -352,9 +352,12 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
         } else {
             iiifQuality = "default.jpg";
         }
-
         if ( levelWidth < tileWidth && levelHeight < tileHeight ){
-            iiifSize = levelWidth + ",";
+            // max will be canonical in 3.0
```

Comment:
Do we need to do anything for backward compatibility for pre-3.0?

Reply:
I don't believe so, the `max` keyword is present in v2... but not in v1. If OSD aims at being compatible with v1 then I can detect the version. I should probably update the PR with that

Reply:
I don't think we've deprecated support for v1; I figure as long as the current code supports it let's try not to break it.

BTW, I see you closed this pull request… Was that intentional? I figure you can just update it rather than making a new one.